app/src/controllers/log/sagas.js

```diff
@@ -127,6 +128,9 @@ function* fetchHistoryItems({ payload } = { payload: {} }) {
   );
 
   yield put(fetchHistoryItemsSuccessAction(response.content));
+  const currentItems = yield select(historyItemsSelector);
+  const loadedItems = currentItems.length - DEFAULT_HISTORY_DEPTH;
+  yield put(setShouldShowLoadMoreAction(loadedItems >= 0));
```

Comment:
We can add additional condition here to avoid unnecessary calculations. As `setShouldShowLoadMoreAction` will be called inside of callback if we get here `loadMore` option:
```
if (!loadMore) {
    const currentItems = yield select(historyItemsSelector);
    const loadedItems = currentItems.length - DEFAULT_HISTORY_DEPTH;
    yield put(setShouldShowLoadMoreAction(loadedItems >= 0));
  }
```